static/js/stream_list.js

```diff
@@ -348,6 +377,7 @@ export function update_dom_with_unread_counts(counts) {
     // counts.stream_count maps streams to counts
     for (const [stream_id, count] of counts.stream_count) {
         set_stream_unread_count(stream_id, count);
+        set_stream_unread_mention_info(stream_id);
```

Comment:
This call, in particular, should access `counts.streams_with_mentions` directly, not call `get_counts` all over again.

Reply:
Implemented it in the style you asked, Thanks for teaching the way to avoid the expensive API call.